### ISSUE TITLE
cell2location not running

### ISSUE THREAD

**Issue by sengao86** [BayraktarLab/cell2location#12]
Hi, 

I am using cell2location now and when I approached Step 4 (Estimating expression signatures Training the model step), the program  is not responding at all sometimes, or frozen at 4% or whatever percentage. I only made one change of the commands as I set 'use_cuda' to False because we don't have GPU equipped.

Hope get some help here.

Thanks!
Sen

**Comment by vitkl** [comment]
Hi @sengao86 

Sorry for the silence - I did not notice the issue until now.

Hm, this is very weird. @yozhikoff this is similar to what we discussed earlier, right? For me, the problem went away after I restarted the jupyter notebook server. It is likely not specific to cell2location but a general problem with tqdm or pytorch.

For 40k cells and 12k genes, the analysis on CPU should take ~24 minutes instead of 3 minutes.
